src/LanguageServer/ProtocolUnitTests/Options/SolutionAnalyzerConfigOptionsUpdaterTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.UnitTests;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Options.UnitTests;
+
+[UseExportProvider]
+public class SolutionAnalyzerConfigOptionsUpdaterTests
```

Comment:
```suggestion
public sealed class SolutionAnalyzerConfigOptionsUpdaterTests
```

Reply:
A test class - we generally don't bother sealing test classes.

Reply:
nit.  feel free to ignore.  i still like sealing personally because some people do subclass tests and it's nice to know.  but feel free to ignore.